src/lib/format/ProtoMsgTsdFormatter.ts

```diff
@@ -58,13 +60,20 @@ export namespace ProtoMsgTsdFormatter {
             return Utility.formatOccupiedName(str);
         });
 
+        if (fileName in WellKnownExtensionsMap) {
+            WellKnownExtensionsMap[fileName]?.declarations?.forEach(declaration => {
```

Comment:
I didn't get here. `fileName` here is the proto file user provided, like the `examples/book.proto`. It will never match the `WellKnownExtensionsMap`.

Reply:
Yes, that's expected! In that scenario, the Google generator will also not match, and therefore those functions will not be present. (The current example does not actually generate descriptors for any well known types.)

Those fields are strictly available when you're translating a well known type with that file extension — see my original PR description for links to where/when this happens in Google protoc. I will add an example here to illustrates, though.